### PR TITLE
fix: base path with nested routes which may have wildcards

### LINKED ISSUE #105: `app.use('/jobs', arena)` + `basePath : '/jobs'` listens on "/jobs/jobs" instead of "/jobs"
As said in the title, I think the routing is incorrect when using this module with nested routes.

Example :

```javascript
const arena = Arena({
  queues: [
    {
      // ...
    }
  ],
}, {
  basePath: '/jobs',
  disableListen: true,
});

// [...]

app.use('/jobs', arena);
// Only reachable by accessing `/jobs/jobs` route
```

When accessing the dashboard on `/jobs/jobs`, the assets can't be accessed and the assets requests results in a 404 error : `/jobs/dashboard.css` (happens with the other assets).

It this an intended behavior ? If so, I didn't find any doc about this :-(
Is this a bug ? If so, [I opened a MR solving this problem](https://github.com/bee-queue/arena/pull/106) in case this is indeed a bug.

Thanks for this great lib by the way !

### PR DESCRIPTION
**Pull request by ronnievdv**
Fixes #105 but also allows for the route to have a wildcard

```
const arena = Arena({
  queues: [
    {
      // ...
    }
  ],
}, {
  basePath: '/jobs',
  disableListen: true,
});

// [...]

app.use('/user/:id', arena);
```

Will result in a path /user/:id/jobs which can then be matched to an actual url like user/1/jobs

### CODE REVIEW

File: src/server/views/layout.hbs [hunk 1]
```diff
@@ -12,16 +12,16 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/bootstrap/3.3.7/css/bootstrap.min.css">
     <!-- JSON Viewer -->
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css">
-    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" nonce="{{ nonce }}"></script>
```
Comment (chuym) [inline src/server/views/layout.hbs]: Could you elaborate a bit more on the purpose of this `nonce`? Is there a warning or similar that requires the use of this?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bull-arena",
+  "name": "@sortdinc/bull-arena",
```
Comment (chuym) [inline package.json]: Please revert this change in the PR

File: package.json [hunk 3]
```diff
@@ -23,5 +23,5 @@
     "node": ">=7.6.0"
   },
   "repository": "https://github.com/bee-queue/arena.git",
-  "version": "2.4.5"
+  "version": "2.4.6"
```
Comment (chuym) [inline package.json]: Please revert this change in the PR

File: src/server/views/layout.hbs (hunk 1 above)
Comment (chuym) [reply src/server/views/layout.hbs]: If you would like to contribute this, please do so in a separate PR with the motivation behind this change. Let's keep this only about #105

**Comment by ronnievdv** [comment]
@chuym  I did a rebase to remove the last two commits, I'll add a separate pull request for adding the nonce.

**Comment by chuym** [comment]
Thank you so much for your fix @ronnievdv ! this is a very wanted fix, will merge now.
